galsim/base.py

```diff
@@ -1053,27 +1188,29 @@ def drawImage(self, image=None, nx=None, ny=None, bounds=None, scale=None, wcs=N
                         it could be useful if you want to view the surface brightness profile of an
                         object directly, without including the pixel integration.
 
+        To convert from flux (photons/cm^2/s) into image values of analog-to-digital units (ADU), we
+        use optional keywords `area` indicating the telescope collecting area in cm^2, `exptime`
+        indicating the exposure time in s, and `gain` which converts between photons and ADU.
```

Comment:
I don't think I have a strong preference here.  I'm not really sure how useful the `added_flux` return value is.  It stems from the desire to know whether the image size is large enough to capture most of the flux.  So you can do a test like `if image.added_flux < 0.99 * obj.flux: ...`  

I guess this means that it should have the same units as obj.flux.